docs/SDK/Explanation/migration.md

```diff
@@ -5,218 +5,211 @@ sidebar_position: 2
 
-This method accepts `iat` and `exp` as optional parameters.
+All of the functionality for obtaining a BLS network signature over a JWT is now implemented by the `getSignedToken` method.
 
-**saveEncryptionKey**
+## Notable Changes
 
-The optional `permanent` parameters are now type `number`.
+V3 includes the following breaking changes from V2.
 
-**saveSigningCondition**
+### Supported Lit Networks
 
-The optional `permanent` parameters are now type `number`.
+Both `jalapeno` and `serrano` will no longer be supported. `cayenne` is the only supported network and will be the new default moving forward.
 
-**signSessionKey**
+### Types
 
-The parameter `chain` is now `chainId` and type `number`.
+#### Removed Types
 
-<br/>
+- `EncryptedString`
+- `ThreeKeys`
+- `JsonStoreSigningRequest`
+- `JsonSaveEncryptionKeyRequest`
 
-### Changed Types
+#### Updated Types
 
-**AccessControlCondition** is now either `AccsRegularParams` or `AccsDefaultParams` and referenced in `AccessControlConditions`.
+- `DecryptZipFileWithMetadata`
+- `VerifyJWTProps`
+- `IJWT`
+- `SuccessNodePromises`
+- `DecryptFromIpfsProps`
+- `LIT_NETWORKS_KEYS`
 
-<br/>
+### Methods
 
-**EVMContractCondition** is now `AccsEVMParams` and referenced in `EvmContractConditions`.
+#### Removed Methods
 
-<br/>
+The following methods have been removed from `LitNodeClientNodeJs` and `LitNodeClient`:
 
-**SolRpcCondition** is now `AccsSOLV2Params` , which now includes more properties like `pdaKey`, `pdaInterface`, and `pdaParams`. `AccsSOLV2Params` is referenced in `SolRpcConditions`.
+- `getChainDataSigningShare`
+- `storeSigningConditionWithNode`
+- `saveSigningCondition`
+- `getSigningShare`
+- `getDecryptions`
+- `getSignedChainDataToken`
 
-<br/>
+#### Updated Methods
 
-**CosmosCondition** is now `ConditionItem`.
+The following methods have their interfaces updated in `LitNodeClientNodeJs` and `LitNodeClient`:
 
-<br/>
+- `combineSharesAndGetJWT`
 
-**ResourceId** is now `JsonSigningResourceId`.
+THe following methods have their names updated in `@lit-protocol/encryption` and `@lit-protocol/lit-node-client-nodejs`:
 
-<br/>
+- `decryptFile` becomes `decryptToFile`
+- `decryptString` becomes `decryptToString`
+- `decryptZip` becomes `decryptToZip`
 
-### Deprecated Methods
+All method interfaces in `@lit-protocol/encryption` have been updated.
 
-These methods are no longer supported:
+## What's Not Migrated?
 
-- signPKPTransaction
-- sendPKPTransaction
-- findLITs
-- sendLIT
-- createHtmlLIT
-- mintLIT
-- unlockLitWithKey
-- toggleLock
-- encryptWithPubKey
-- decryptWithPrivKey
-- lookupNameServiceAddress
-- metadataForFile
-- configure
+All of the access control conditions that have been "stored" in the `jalapeno` and `serrano` networks **will not be migrated** but we will continue to maintain support for them. If you wish to continue using these access control conditions for encryption or signing JWTs, please continue using the V2 SDK.
```

Comment:
maybe consider moving this info to the top of the page since it's very important